app/screens/settings/index.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+import {of as of$} from 'rxjs';
+import {switchMap} from 'rxjs/operators';
+
+import {observeConfig} from '@queries/servers/system';
+import {isValidUrl} from '@utils/url';
+
+import Settings from './settings';
+
+import type {WithDatabaseArgs} from '@typings/database/database';
+
+const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
+    const config = observeConfig(database);
+    const siteName = config.pipe(switchMap((c) => of$(c?.SiteName) || of$('Mattermost')));
+    const showHelp = observeConfig(database).pipe(switchMap((c) => of$(c?.HelpLink ? isValidUrl(c.HelpLink) : of$(false))));
```

Comment:
I may be wrong (too many nested parenthesis) but it seems that showHelp is of type `Observable<boolean|Observable<boolean>>`. What I mean is that the second `of$` should not be needed, right? Either we use one "global" `of$` before the ternary operator (as you do with the first `of$`) or we add one `of$ in both results of the ternary operator.

I think the "global" version works better, but up to you.

Reply:
You're right.  Updated it.